blog/_posts/2025-04-22-Enhance-security-OS-Security-Patch.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Enhancing Your Operating System's Security with OS Security Patches in AKS"
+description: "Learn the what, the whys, and the hows of configuring OS Security Patch Auto upgrade channel.  The article also covers some reasons for OS Security Patch, with some primary benefits being increased performance, better security, and minimal disruption to workloads."
+date: 2025-04-22
+authors:
+- Kaarthikeyan Subramanian
+- Ye Wang
+categories: 
+- Security
+- Operations
+---
+
+
+## Traditional patching and the need for Managed patching.
+
+Operating System (OS) security patches are critical for safeguarding systems against vulnerabilities that malicious actors could exploit. These patches help ensure your system remains protected against emerging threats. Traditionally, customers have relied on nightly updates, such as [unattended upgrades in Ubuntu](https://help.ubuntu.com/community/AutomaticSecurityUpdates) or [Automatic Guest OS Patching](https://learn.microsoft.com/en-us/azure/virtual-machines/automatic-vm-guest-patching) at the virtual machine (VM) level. However, when kernel security packages were updated, a host machine reboot was often required, typically managed using tools like [kured](https://github.com/kubereboot/kured). 
+
+This approach, while effective, introduced challenges. Untested or uncontrolled security packages occasionally caused outages, emphasizing the need for a more reliable and managed patching mechanism at the node level. Additionally, maintaining reboot daemonsets like [kured](https://github.com/kubereboot/kured) added operational overhead for many customers. Ideally, customers prefer managed Kubernetes services, such as Azure Kubernetes Service (AKS), to handle OS security patching comprehensively and seamlessly.
+
+## Automatic Node OS Security Patching mechanisms at AKS
+
+AKS provides two managed and tested mechanisms to deliver the latest security packages to your Node Operating System. Learn more about [OS Security Patch](https://learn.microsoft.com/azure/aks/auto-upgrade-node-os-image?tabs=azure-cli).
```

Comment:
Link at the end seems out of place? Should be one link at the end of each of the below bullets maybe?